--- a/runtime/classes/propel/util/BasePeer.php	2018-05-30 14:27:10.799698441 +0200
+++ b/runtime/classes/propel/util/BasePeer.php	2018-05-30 14:27:38.903159425 +0200
@@ -353,8 +353,13 @@
 
 			$stmt = null;
 			try {
-
-				$sql = "UPDATE " . $tableName . " SET ";
+				$sql = "UPDATE ";
+                                if ($db->useQuoteIdentifier()) {
+                                        $sql .= $db->quoteIdentifierTable($tableName);
+                                } else {
+                                        $sql .= $tableName;
+                                }
+                                $sql .= " SET ";
 				$p = 1;
 				foreach ($updateTablesColumns[$tableName] as $col) {
 					$updateColumnName = substr($col, strrpos($col, '.') + 1);
